generators/python/sdk/CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.12.4] - 2024-03-19
+
+- Improvement: Allow full forward compat with enums while keeping intellisense by unioning enum literals with typing.AnyStr.
```

Comment:
nit: can we add the before and after signature in a python code snippet 🙏

Reply:
```python
# Before
Color = typing.Literal["red", "blue"]

# After
Color = typing.Union[typing.AnyStr, typing.Literal["red", "blue"]]
```

Reply:
updated!